internal/aggregator/utils.go

```diff
@@ -23,21 +24,30 @@ func extractJobSetAttrs(js *jobset.JobSet) records.Attrs {
 		//   cloud.google.com/gke-tpu-topology: 2x2x1
 		//   cloud.google.com/gke-spot: "true"
 		//
+		rjChipCount := 0
```

Comment:
No need for this var, just increment `chipCount` directly

Reply:
Fixed in a8ed93cd59d7d55ed973e046463ea7b59a410b51